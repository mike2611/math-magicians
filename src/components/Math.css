.math {
  justify-content: space-around;
} 
