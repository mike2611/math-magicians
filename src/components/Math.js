import Calculator from './Calculator';
import './Math.css';

const Math = () => (
  <div className="math">
    <h1>Let&apos;s do some math!</h1>
    <Calculator />
  </div>
);

export default (Math);
