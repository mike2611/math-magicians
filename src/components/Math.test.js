import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Math from './Math';

describe('Math component tests', () => {
  test('Should render Home component correctly' , () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Math />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });

  test("Renders Let's do some math! as an h1", () => {
    const component = render(<Math />);
    component.getByText("Let's do some math!");
  });
});
