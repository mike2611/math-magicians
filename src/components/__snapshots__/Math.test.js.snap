// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Math component tests Should render Home component correctly 1`] = `
<div
  className="math"
>
  <h1>
    Let's do some math!
  </h1>
  <Calculator />
</div>
`;
